docs/usecases/testing.md

```diff
@@ -72,3 +72,271 @@ Test aspects are powerful tools for modifying behavior of individual tests or ev
 **Zero Dependencies**
 
 As a library with zero third party dependencies, this project is available on the JVM, ScalaJS, and will be available on Dotty and Scala Native in the near future. So you can write your tests once and make sure that your code works correctly across all platforms that you support.
+
+
+**How zio-test was designed**
+
+`zio-test` is designed around the idea of making test first-class objects. What it means is that tests 
```

Comment:
I think this is definitely right. I might say that not only are tests values, they are also effects. This has a couple of advantages. First, we solve the asynchrony problem where in other frameworks you have to "run" your effects to test them, but then you can't block, especially on Javascript, so then you basically need to rewrap them in some new effect or quasi-effect type like `Future`, creating a lot of indirection. Second, making tests effects gives us access to all the tools of `ZIO` which lets use solve lots of problems in very simple and straightforward ways. Want a timeout on your tests? You don't need special logic. `ZIO` already has a `timeout` method`. Retry a test a bunch of times to make sure it isn't flaky? `repeat`? Use some type of fixture? `Managed`.

Reply:
👍